src/models/projectOptions.ts

```diff
@@ -12,5 +14,11 @@ export type NodeOptions = {
     architecture: Architecture;
 };
 
+export type PythonOptions = {
+    pythonRuntime: PythonRuntime;
+    architecture: Architecture;
+};
+
 export const supportedNodeRuntimes = ["nodejs20.x"] as const;
 export const supportedArchitectures = ["arm64", "x86_64"] as const;
+export const supportedPythonRuntimes = ["python3.13.x"] as const;
```

Comment:
We should more flexible with python versions because python releases more often than node (almost yearly) and also I believe users don't update often (I have python 3.10 installed on my machine).

As a rule of thumb we should follow the Python releases cycle [1]. I would be happy to as flexible as possible for our users and don't force them to upgrade their python - so I would support >= 3.9.

[1] https://devguide.python.org/versions/